.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -115,84 +115,6 @@ jobs:
           cmake --build . --config Release --parallel 4
         working-directory: wamr-compiler
 
-  build_iwasm_linux_gcc4_8:
```

Comment:
The building with gcc-4.8 is also moved to nightly build, right? @loganek, is it OK for you?

Reply:
yeah, I think that's right thing to do

Reply:
I think it's fine to move it to nightly.